app/models/graded_offspring.rb

```diff
@@ -1,2 +1,3 @@
 class GradedOffspring < Offspring
+  validates :grade, presence: true, numericality: { only_integer: true, greater_than: 0 }
```

Comment:
something like this? 
`  validates :age, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 1,
    less_than_or_equal_to: 99 }`